fem/bilinearform_ext.cpp

```diff
@@ -765,6 +765,197 @@ void PABilinearFormExtension::MultTranspose(const Vector &x, Vector &y) const
    }
 }
 
+void PABilinearFormExtension::AbsMult(const Vector &x, Vector &y) const
```

Comment:
Is it possible to reduce the code duplication here and in similar functions? Maybe `PABilinearFormExtension::Mult` and `PABilinearFormExtension::AbsMult` can call a common function where most of the code is with an extra parameter?

Reply:
I think so. It is related to the comment below.

Reply:
If we can reduce code duplication, I think we should

Reply:
Agreed! Although I have a concern with classes derived from `Operator`, as `Operator::Mult` is purely virtual. Adding a parameter implies overloading the `Derived::Mult` function (different signature will not override `Operator::Mult(const Vector&, Vector&)`).
Even if `Derived::Must(const Vector&, Vector&, bool ABS = false)` is defined, it would create ambiguous calls. Right?

Reply:
I think what Tzanio is suggesting is the following: `PABilinearFormExtension::Mult` and `PABilinearFormExtension::AbsMult` both call a new member function (with a placeholder name for now) `PABilinearFormExtension::MultCommon` that takes an additional boolean parameter. `Operator::Mult` and its overridden versions in derived classes would retain the same signature.

Reply:
Ok, that makes sense! Thank you